src/app/components/article/MediaArticlesTeamArticles.js

```diff
@@ -94,10 +133,18 @@ const MediaArticlesTeamArticles = ({
       render={({ error, props }) => {
         if (!error && props) {
           // Merge explainers with fact-checks and re-sort
-          const articles = props.team.factChecks.edges.concat(props.team.explainers.edges).map(edge => edge.node).sort((a, b) => (parseInt(a.created_at, 10) < parseInt(b.created_at, 10)) ? 1 : -1);
+
+          let articles = [];
+          let hasRelevantArticles = false;
+          if (props.project_media.relevant_articles_count > 0) {
+            hasRelevantArticles = true;
+            articles = props.project_media.relevant_articles.edges.map(edge => edge.node);
+          } else {
+            articles = props.team.factChecks.edges.concat(props.team.explainers.edges).map(edge => edge.node).sort((a, b) => (parseInt(a.created_at, 10) < parseInt(b.created_at, 10)) ? 1 : -1);
```

Comment:
Suggestion:  might be helpful to have a comment here about why the fallback logic is used? i.e "if there are no relevant articles, we assume recent articles are returned and should be sorted by recency"